mrp_food/models/product_product.py

```diff
@@ -34,10 +34,19 @@ class ProductProduct(models.Model):
     # because the computation is based on mrp.bom.line,
     # that is not available for user that doesn't belong to mrp user group
     is_component = fields.Boolean(
-        compute="_compute_is_component",
+        compute="_compute_is_component_intermediate",
         compute_sudo=True,
         store=True,
-        default=False,
+        help="Is component a product which has no BoM based on it +\
+              not to sale + is used in a BoM line",
+    )
+    is_intermediate = fields.Boolean(
+        string="Intermediate product",
+        compute="_compute_is_component_intermediate",
+        compute_sudo=True,
+        store=True,
+        help="Is intermediate a product which has a BoM based on it +\
+              is or is not to sale + is used in a BoM line",
```

Comment:
tu parles de to sale (or not to sale, that's the question); mais je vois pas de ``if product.sale_ok`` dans le champ compute. je me trompe ?

Reply:
Yes ya pas car justement on s'en fout de sale_ok.
Du coup je pourrais ne pas le dire mais c'est pour expliciter que ce n'est pas un critère 
P-e que ça rend pas clair le truc du coup !